               # request is empty
---
bool wallfound # response 